### ISSUE TITLE
[Crash] Ars Nouveau 1.15.0 causes crash on world load

### ISSUE THREAD

**Issue by faeldray** [MintTeaTx/PMMOBridge#2]
The latest version of Ars Nouveau along with PMMO Bridge causes Minecraft to crash on world load.

https://pastebin.com/BB2cCvMd

**Comment by litjohn50** [comment]
Same issue. Anything that has ars nouveau crashes you. Wether its enchants on your armor just taking it off.

https://gist.github.com/litjohn50/c5957acc6fc10de70d0edf0c14d60aa1

**Comment by AngellusMortis** [comment]
Issue caused by this change: https://github.com/baileyholl/Ars-Nouveau/commit/16c8b622f9d9a8c94707faefbbd4a1eb8e8ff296#diff-f2d238020192ab22e0fb5f87f1b77197c86eaf0c961563cc1c51d62a23734d7e

Mod should just need recompiled with new version of Ars Nouveau + remove the `int` cast.

Workaround for now is to downgrade Ars Nouveau to [1.14.1](https://www.curseforge.com/minecraft/mc-mods/ars-nouveau/files/3286983)

**Comment by starluneaux** [comment]
Yeah I've been working on it.  The problem is that Ars swapped to official mappings while PMMO is on the unofficial, breaking Mixins

**Comment by starluneaux** [comment]
Not to day it's the only problem, I've just been busy both moving across the state and trying to fix that issue preventing me from compiling

**Comment by XxInvictus** [comment]
I didn't want to post this earlier as I didn't want to hijack the project and I thought it would be resolved sooner, but here is a local copy of mine.

I want to make the disclaimer that **I do not know if this works as intended**, I tried to add PMMO Bridge when it was broken so I never used the working version. In saying that, this version does not cause the aforementioned error.
[pmmobridge-1.16.4-1.0.4.jar.zip](https://github.com/MintTeaTx/PMMOBridge/files/6690091/pmmobridge-1.16.4-1.0.4.jar.zip)

**Comment by XxInvictus** [comment]
Basically in the above version I have updated the gradle to include a newer version of PMMO and I have replaced the int castings within SetRegen with double castings to reflect the PMMO change.

**Comment by XxInvictus** [comment]
It's been a long time coming but I have just uploaded a new version to curseforge which should fix this bug:
https://www.curseforge.com/minecraft/mc-mods/pmmo-bridge/files/3640552
